IS0401Test.py

```diff
@@ -82,40 +90,52 @@ def do_registry_basics_prereqs(self):
         if DNS_SD_MODE == "multicast":
             registry_mdns = []
             priority = 0
+
+            # Add advertisement with invalid version
+            info = self._registry_mdns_info(self.invalid_registry.get_data().port, priority, "v9.0")
+            registry_mdns.append(info)
+            # Add advertisement with invalid protocol
+            info = self._registry_mdns_info(self.invalid_registry.get_data().port, priority, None, "invalid")
+            registry_mdns.append(info)
+
+            # Add advertisement for primary and failover registries
             for registry in self.registries:

```

Comment:
Where has the construction of the `registry_mdns` info for the primary registry gone, now it's been separated out from the others by line 40?

Reply:
Oh no, my mistake, it's only the invalid registry that's been removed from the passed in `registries`.